services/notification/notifiers/comment/conditions.py

```diff
@@ -154,7 +154,10 @@ async def _check_coverage_drop(comparison: ComparisonProxy) -> bool:
         project_status_config = read_yaml_field(
             comparison.comparison.current_yaml, ("coverage", "status", "project"), {}
         )
-        threshold = Decimal(project_status_config.get("threshold", 0))
+        threshold = 0
+        if isinstance(project_status_config, dict):
+            # Project status can also be a bool value, so check is needed
+            threshold = Decimal(project_status_config.get("threshold", 0))
```

Comment:
What does it mean for threshold to be a bool? If set to True, what value does it use as a threshold?

Reply:
If the project status is a bool it means "I want project status with the default configuration". The default threshold is 0.

Reply:
sounds good, ty for clarifying